Lib/glyphsLib/builder/context.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import (print_function, division, absolute_import,
+                        unicode_literals)
+
+from collections import OrderedDict
+
+
+class GlyphsToUFOContext(object):
+    """Builder context for Glyphs to UFO + designspace."""
+
+    def __init__(self, font, defcon):
+        """Create a context for the Glyphs to UFO + designspace builder.
```

Comment:
I was thinking, instead of passing around a "context" object as argument to the `to_ufo_*` and `to_glyphs_*` functions, we could define two classes e.g. `UFOBuilder' and `GlyphsBuilder` that have those functions as instance methods and use `self` as the context.

We could still keep this modularized structure with the builder functions in their respective modules, and then when we define the builder classes, we would set these functions as class attributes, and they will be bound to a builder instance just like regular methods.

WDYT?

Reply:
Using classes makes sense.

Reply:
Yes, I will do that